src/deploy_tools/models/binary_app.py

```diff
@@ -0,0 +1,43 @@
+from enum import StrEnum
+from typing import Literal
+
+from pydantic import Field
+
+from .parent import ParentModel
+
+
+class HashType(StrEnum):
+    """
```

Comment:
Single line summaries are normally in-line with the quotes:
```
"""Type of hash to use for the binary download."""
```

Reply:
fixed.